packages/template-retail-react-app/app/components/product-view-modal/bundle.jsx

```diff
@@ -52,7 +55,11 @@ const BundleProductViewModal = ({product: bundle, isOpen, onClose, updateCart, .
                 <ModalBody pb={8} bg="white" paddingBottom={6} marginTop={6}>
                     <Flex direction={['column', 'column', 'column', 'row']}>
                         {/* Due to desktop layout, we'll need to render the image gallery separately, from outside the ProductView */}
-                        <Box flex={1} mr={[0, 0, 0, 6, 6]}>
+                        <Box
+                            flex={1}
+                            mr={[0, 0, 0, 6, 6]}
+                            display={['none', 'none', 'none', 'block']}
```

Comment:
As the comment on line 57 indicates, this standalone image gallery is meant for _desktop_ layout use only.